.buildkite/scripts/validate-tag.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -euo pipefail
+
+source .buildkite/scripts/tooling.sh
+
+transformTagAndValidate() {
+    version=$1
+    version=${version//\v/}
+    if [[ $version =~ ^[0-9]+.[0-9]+.[0-9]+(-[A-Za-z0-9_]+)?$ ]]; then
+        echo "valid version: ${version}"
+    else
+        echo "unvalid version: ${version}"
+        echo "unsupported docker tag, please use the major.minor.path(-prerelease)? format (for example: 1.2.3 or 1.2.3-alpha)."
+        exit 1
+    fi
+}
+
+DOCKER_TAG_VERSION=$(buildkite-agent meta-data get DOCKER_TAG_VERSION)
+transformTagAndValidate "$DOCKER_TAG_VERSION"
+buildkite-agent meta-data set DOCKER_TAG_VERSION "${DOCKER_TAG_VERSION}"
```

Comment:
Does this update the value of that environment value for following steps in the pipeline? 

Did you try to use as a version `v8.8.1` for instance ? 

That value would be the one set by the job that is going to trigger this. For instance, this is an example of build trigger in Jenkins:
- `DOCKER_TAG: v8.8.0` https://fleet-ci.elastic.co/view/All/job/release-package-registry-distribution/13/parameters/